worker/chaos-worker.go

```diff
@@ -109,19 +110,12 @@ func (w *Worker) CheckMinioHealth(args *string, reply *struct{}) error {
 // And then verify whether starting Minio was succesfull.
 // If not just verify whether minio server is running.
 // A `nil` error response indicates the master that the worker and Minio server is running on the specified port.
-func (w *Worker) InitChaosWorker(args *shared.MinioNode, reply *struct{}) error {
-	// Start Minio server if `-start-minio` flag is enabled.
-	if args.StartMinio {
-		// Start Minio server.
-		err := StartMinio()
-		// return error for the RPC call.
-		if err != nil {
-			return err
-		}
-	}
+func (w *Worker) InitChaosWorker(args *struct{}, reply *struct{}) error {
 	log.Println("Initializing the Node for the Chaos test.")
 	// Verifies whether Minio is running on the specified port.
-	err := IsMinioRunning(args.Addr)
+	addr := "http://127.0.0.1:9000"
```

Comment:
No reason to set the 'addr' then. Move it as argument to 'IsMinioRunning("http://127.0.0.1:9000")` directly. Also what happens if the Minio server is running on non-standard port?. Since this is a controller env i don't expect such things to occur.